core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.spec.ts

```diff
@@ -40,17 +40,17 @@ describe('Oppotunities backend API serice', function() {
     SkillOpportunityObjectFactory = null;
   let urlInterpolationService:
     UrlInterpolationService = null;
-  const skillResponse = {
+  const skillOpportunityResponse = {
     opportunities: [{
       id: 'skill_id',
       skill_description: 'A new skill for question',
       topic_name: 'A new topic',
       question_count: 30
     }],
     next_cursor: '6',
-    more: ''
+    more: true
   };
-  const sampleExplorationSumary = {
+  const skillExplorationOpportunity = {
```

Comment:
Drop the 'Exploration'. SkillOpportunities aren't really related to explorations.

Reply:
Done!